core/src/main/java/tc/oc/pgm/destroyable/Destroyable.java

```diff
@@ -356,12 +356,15 @@ public DestroyableHealthChange handleBlockChange(
             NMS_HACKS.skipFireworksLaunch(firework);
           }
 
-          // Players more than 64m away will not see or hear the fireworks, so just play the sound
-          // for them
-          for (MatchPlayer listener : this.getOwner().getMatch().getPlayers()) {
-            if (listener.getBukkit().getLocation().distance(blockLocation) > 64) {
-              listener.playSound(Sounds.OBJECTIVE_FIREWORKS_FAR);
-              listener.playSound(Sounds.OBJECTIVE_FIREWORKS_TWINKLE);
+          if (this.definition.hasSparksMatch()) {
+            // Players more than 64m away will not see or hear the fireworks, so play sound for them
+            for (MatchPlayer listener : this.getOwner().getMatch().getPlayers()) {
+              if (listener.getBukkit().getLocation().distance(blockLocation) > 64) {
+                listener.playSound(
+                    sound(key("fireworks.blast_far"), Sound.Source.MASTER, 0.75f, 1f));
```

Comment:
There's a reason all sounds are defined in the Sounds class, please follow the standard so multi-version works

Reply:
Hmm, I didn't change anything with respect to sound, so don't really understand the build error.

How did this go wrong?

Reply:
You switched it from `Sounds.OBJECTIVE_FIREWORKS_FAR` to `sound(key("fireworks.blast_far"), Sound.Source.MASTER, 0.75f, 1f)`. We put all sounds constants in [Sounds](https://github.com/PGMDev/PGM/blob/dfa78531f9364c1194632ae6f4b453884b709548/util/src/main/java/tc/oc/pgm/util/bukkit/Sounds.java) since the sound keys are not version agnostic.